src/api.c

```diff
@@ -10,6 +10,7 @@
 
 	#include <unistd.h>
 	#include <sys/syscall.h>
+	#include <asm-generic/unistd.h>
```

Comment:
To minimize possible conflicts for systems where inclusion of `unistd.h` already defines `__NR_getcpu`, I suggest:
```
	#if !defined(__NR_getcpu)
		#include <asm-generic/unistd.h>
	#endif
```
With this change, I confirm that pytorch master builds successfully here (Ubuntu 18.04, using conda environment).

Reply:
Thanks.  Regarding your question on what PyTorch tests to run - it is difficult to say.  cpuinfo is used both directly inside PyTorch and by its dependencies.  Many codepaths in PyTorch are conditionally enabled or disabled with CMake variables and preprocessor directives in the code, and there are also different platforms to consider so running the CI in its entirety is a must in my opinion.